async-nats/src/jetstream/context.rs

```diff
@@ -49,4 +54,25 @@ impl Context {
 
         Ok(response)
     }
+
+    pub async fn add_stream<S>(&mut self, stream_config: S) -> Result<StreamInfo, Error>
+    where
+        StreamConfig: From<S>,
+    {
+        let config: StreamConfig = stream_config.into();
+        if config.name.is_empty() {
```

Comment:
Lets ditch this check and add a test for it, ensure the server does still error.

```suggestion
```

Reply:
as found out, it returns `no responders`. Im still all for server-checks, but in this case... I would have this in.